pylib/mozhginfo/setup.py

```diff
@@ -0,0 +1,18 @@
+from setuptools import setup, find_packages
+
+setup(
+    name='pushlog_client',
+    version='0.1',
+    packages=find_packages(),
+
+    install_requires=[
+        'requests>=2.5.1'
+    ],
+
+    # Meta-data for upload to PyPI
+    author='Armen Zambrano G.',
+    author_email='armenzg@mozilla.com',
+    description="Script designed to query information about who pushes what where and when.",
+    license='MPL2',
+    url='https://github.com/armenzg/version-control-tools/pylib/mozhginfo',
```

Comment:
Could you please use this URL?
https://hg.mozilla.org/hgcustom/version-control-tools/file/tip/pylib/mozhginfo